service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java

```diff
@@ -232,22 +250,27 @@ private Set<String> getRepresentativesEmails(CaseData caseData,
     @Async
     public void notifyEmailServedRepresentatives(final AdditionalApplicationsUploadedEvent event) {
         final CaseData caseData = event.getCaseData();
-        NotifyData notifyData = contentProvider.getNotifyData(caseData);
 
-        Set<String> emailRepresentatives = getRepresentativesEmails(caseData, EMAIL);
+        AdditionalApplicationsBundle bundleUploaded = caseData.getAdditionalApplicationsBundle().get(0).getValue();
```

Comment:
it assumes the uploaded bundle is at the index 0, right?

Reply:
yes, following existing behaviour